client/src/pages/emails/Show.js

```diff
@@ -0,0 +1,76 @@
+import { gql } from "@apollo/client"
+import { DEFAULT_PAGE_PROPS, DEFAULT_SEARCH_PROPS } from "actions"
+import API from "api"
+import {
+  mapPageDispatchersToProps,
+  PageDispatchersPropType,
+  useBoilerplate,
+  usePageTitle
+} from "components/Page"
+import moment from "moment"
+import React from "react"
+import { Table } from "react-bootstrap"
+import { connect } from "react-redux"
+import Settings from "settings"
+
+const GQL_GET_EMAILS = gql`
+  query {
+    allEmails: pendingEmails {
```

Comment:
I would simply put `pendingEmails` here, and on line 54 use `data.pendingEmails.map(…)`.